native-stack/views/HeaderConfig.tsx

```diff
@@ -73,6 +79,14 @@ export default function HeaderConfig(props: Props) {
       {headerRight !== undefined ? (
         <ScreenStackHeaderRightView>{headerRight()}</ScreenStackHeaderRightView>
       ) : null}
+      {headerLeft !== undefined ? (
```

Comment:
The reason these were not exposed is because of inconsistent behaviour between Android and iOS. Can you check if it still happens?